File: Andriod/Constructor/construction/src/main/java/com/palitri/openiot/construction/framework/composite/CompositeProperty.java
package com.palitri.openiot.construction.framework.composite;

import com.palitri.openiot.construction.framework.board.models.BoardProperty;
import com.palitri.openiot.construction.framework.web.models.configurations.project.Peripheral;
import com.palitri.openiot.construction.framework.web.models.configurations.project.PeripheralProperty;

public class CompositeProperty {
    public BoardProperty boardProperty;
    public PeripheralProperty peripheralProperty;
    public Peripheral parentPeripheral;
}

File: Windows/OpenIoT.App/OpenIoT.Lib/Composite/CompositeProperty.cs
using Palitri.OpenIoT.Board.Models;
using Palitri.OpenIoT.Web.Models.Configurations.Project;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Palitri.OpenIoT.Composite
{
    public class CompositeProperty
    {
        public BoardProperty BoardProperty { get; set; }
        public PeripheralProperty PeripheralProperty { get; set; }
        public Peripheral ParentPeripheral { get; set; }
        public CompositeBoard Board { get; set; }
    }
}
